wgsl/index.bs

```diff
@@ -4383,10 +4445,10 @@ For details on conversion to and from floating point types, see [[#floating-poin
       <td>Conversion of a boolean value to an unsigned integer<br>
           The result is 1u if |e| is true and 0u otherwise.
   <tr algorithm="scalar conversion from signed integer to unsigned integer">
-      <td>|e|: i32<td>`u32(`|e|`)`: u32
+      <td>|e|: AbstractInt or i32<td>`u32(`|e|`)`: u32
       <td>Reinterpretation of bits.<br>
-          The result is the unique value in [=u32=] that is equal to (|e| mod 2<sup>32</sup>).
-  <tr algorithm="scalar conversion from binary32 floating point to unsigned integer">
+          The result is the unique value in [=u32=] that has the same bit pattern as |e|.
```

Comment:
I guess AbstractInt values have a bit pattern, because we define bitwise operations on them.

This rephrasing would make it an error to try to stuff a too-big AbstractInt value into a u32.  It would be overflow. Similarly for a negative abstract-int value.  That makes sense to me.